DEVELOPMENT.md

```diff
@@ -0,0 +1,178 @@
+6. Telling `npm` how to talk to GitHub Packages
+   1. Generating a personal access token
+   2. Giving that personal access token to `npm`
+<!-- 7. Installing all of the third-party libraries -->
+
+## Creating a GitHub Account
+
+The code for BuzzAbout&mdash;along with its revision history&mdash;is stored on [GitHub][], a website for hosting software projects. To work on Buzz About, you're going to need to sign up for a GitHub account [here][Sign up for GitHub].
+
+![GitHub's "sign up" page][i:Signing up for GitHub]
+
+You'll want to join the [Minnesota Pollinators][] organization ([Prof. KK Lamberty][] can invite you).
+
+Be sure to sign up for the [student developer pack][Student developer pack], as well!
+
+![The student developer pack's webpage][i:Student developer pack]
+
+## Installing Visual Studio Code
+
+Next, in order to read and write the code, you're going to need a text editor. There are a lot of good options, but the one that we've been using is [Visual Studio Code][], particularly because of its [live share][Live share] feature.
+
+![Visual Studio Code][i:VS Code]
+
+## Installing `git`
+
+`git` is a version-control system: it keeps track of all of the changes made to the project, and lets several people work on the same code in parallel.
+
+### On Windows
+
+If you're on Windows, you can go [here][Install git for Windows] to download `git`.
+
+When the installer asks you to choose the default editor, you may want to select Visual Studio Code. Other than that, the default options should be just fine.
+
+![The git installer program on Windows, asking what the default editor should be, with "Visual Studio Code" selected][i:Windows, installing git]
+
+### On a Mac
+
+If you're on a Mac, you can get `git` as part of the XCode command-line developer tools. Open up Terminal.app and type `git --version`:
+
+<img width="638" alt="Terminal.app with the command &quot;git --version&quot; typed" src="https://user-images.githubusercontent.com/56209343/89746736-f0de7000-da80-11ea-9ab5-cf48acd6b5f7.png">
+
+You'll get a pop-up asking you to install the developer tools:
+
+<img width="573" alt="The pop-up window asking if you'd like to install the tools" src="https://user-images.githubusercontent.com/56209343/89746735-f0de7000-da80-11ea-9075-abcefdc23124.png">
+
+If you don't get a pop-up, but instead get a version number like this, that means `git` is already installed.
+
+<img width="682" alt="Terminal.app showing the command &quot;git --version&quot; and the output &quot;git version 2.24.3 (Apple Git-128)&quot;" src="https://user-images.githubusercontent.com/56209343/89746734-f045d980-da80-11ea-8944-f2b0de3b2ecb.png">
+
+## Installing GitKraken
+
+`git` by itself is very spare, so we've been using it through a program called [GitKraken][]. (This program has a nice chart showing all of the changes over time.) You can use GitKraken for free, but you actually have access to the Pro version through GitHub's student developer pack.
```

Comment:
I don't think we should assume the reader is a student.

Reply:
Hmm, okay. What do you think about phrasing it like "&hellip; but if you're a student at Morris, you can access to the Pro version through &hellip;"?